### PR TITLE
Save nunjuck env reference in express app settings

### PR DESCRIPTION
**Pull request by lukechilds**
Resolves https://github.com/mozilla/nunjucks/issues/828

### CODE REVIEW

File: src/environment.js [hunk 1]
```diff
@@ -286,6 +286,7 @@ var Environment = Obj.extend({
         };
 
         app.set('view', NunjucksView);
+        app.set('nunjuckEnv', this);
```
Comment (carljm) [inline src/environment.js]: Its nunjucks, with the `s`, so this should be `nunjucksEnv`

**Comment by carljm** [comment]
Can you add a test for this?

File: src/environment.js (hunk 1 above)
Comment (lukechilds) [inline src/environment.js]: Sorted 👍

**Comment by lukechilds** [comment]
I've added what I think should be a passing test but I'm not quite sure how to run the express tests. `npm test` doesn't seem to do anything with that file and I can't find anything documenting it. Any ideas?

**Comment by carljm** [comment]
Huh, looks like the express tests aren't running. Odd. I'll create a separate issue for that. Test looks fine, anyway, I'll go with it for now.

**Comment by lukechilds** [comment]
@carljm Yeah, it definitely works, I've manually copied this version into `node_modules` and it's working great.

Don't mean to rush you, sure you're busy, but just wondering if you had any plans to release this on npm soon? I'm currently working on a time sensitive project and it would be awesome to be able to get rid of my current workaround from the issue I posted.

**Comment by carljm** [comment]
I'll try to make a release soon.

**Comment by lukechilds** [comment]
@vecmezoni great to see you've stepped up to help maintain this project. I just noticed you've published a new build to npm. Is there any chance you could publish a version with this PR in?

**Comment by vecmezoni** [comment]
@lukechilds ok, tomorrow i'll check 2.x branch and publish 2.5.0 😉

**Comment by lukechilds** [comment]
That would be amazing, thank you! 🍻

**Comment by vecmezoni** [comment]
@lukechilds can't wait till tomorrow, published 2.5.0 with this PR.

**Comment by lukechilds** [comment]
@vecmezoni working perfectly, cheers.

**Pull request merged**